### ISSUE TITLE
API Fails on default skins (TypeError: Cannot read property 'isEmpty' of undefined)

### ISSUE THREAD

**Issue by zikeji** [Electroid/mojang-api#26]
```
TypeError: Cannot read property 'isEmpty' of undefined
    at w (worker.js:15:150720)
    at async i (worker.js:15:158788)
```

The API endpoint fails when you try and query a user that doesn't have a skin (e.g. a default skin).

**Comment by Electroid** [comment]
Can you give a username or uuid so I can reproduce?

**Comment by zikeji** [comment]
Yes - this is my particular user who ran into the issue.

https://api.ashcon.app/mojang/v2/user/Asi3nF3si0nYT

**Comment by Electroid** [comment]
This is very odd, I have a feeling Mojang has changed their API in some way, I'm investigating. (The code powering my API has not changed)

**Comment by zikeji** [comment]
Yeah. I tried running your code to debug it myself but couldn't get the master branch running properly. Not sure if that's the branch I should be testing on.

**Comment by Electroid** [comment]
What's on `master` updated and a fix was just pushed globally.

Basically Mojang changed the default response from `sessionserver.mojang.com`, so I had to add some extra nil guards. Fixed in 27867efb037106b28a26fa89a42c9fe19328e393
